### PR TITLE
Fixes DropDown runtime select list

### PR DESCRIPTION
**Pull request by noxe**
fixes https://github.com/elsa-workflows/elsa-core/issues/2531 

using items instead selectlist

### CODE REVIEW

**Comment by Vendetta26** [comment]
gh pr checkout 2532

**Comment by Vendetta26** [comment]
requests

File: src/designer/elsa-workflows-studio/src/components/editors/properties/elsa-dropdown-property/elsa-dropdown-property.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ export class ElsaDropdownProperty {
     const fieldId = propertyName;
     const fieldName = propertyName;
     let currentValue = this.currentValue;
-    const {items} = this.selectList;
+    const { items } = this.selectList.items;
```
Comment (sfmskywalker) [inline src/designer/elsa-workflows-studio/src/components/editors/properties/elsa-dropdown-property/elsa-dropdown-property.tsx:51]: This should not work - if it does, it means the contents of the `this.selectList` contains an unexpected value (an object containing `items` which itself then also has an `items` field.

I just pushed an update that seems to fix the runtime select list items story (I tried it with a sample custom activity with runtime items implemented).

Please give it a go and let me know if it works or not.

Thanks!

Comment (noxe) [reply src/designer/elsa-workflows-studio/src/components/editors/properties/elsa-dropdown-property/elsa-dropdown-property.tsx:51]: ok - just did a compare with other implementations - and this was the only where this.selectlist was used - the other editor are using selectlist.items.

Comment (noxe) [reply src/designer/elsa-workflows-studio/src/components/editors/properties/elsa-dropdown-property/elsa-dropdown-property.tsx:51]: and btw - the webservice call worked - i got a javascript exception - but i try again once the build is finished

image

Comment (noxe) [reply src/designer/elsa-workflows-studio/src/components/editors/properties/elsa-dropdown-property/elsa-dropdown-property.tsx:51]: works perfect now - sorry for my wrong assumption :)

**Pull request closed without merging**